rpc_api.go

```diff
@@ -55,8 +55,8 @@ func (rpcapi *RPCAPI) PinPath(ctx context.Context, in *api.PinPath, out *api.Pin
 }
 
 // UnpinPath resolves path into a cid and runs Cluster.Unpin().
-func (rpcapi *RPCAPI) UnpinPath(ctx context.Context, in string, out *api.Pin) error {
-	pin, err := rpcapi.c.UnpinPath(ctx, in)
+func (rpcapi *RPCAPI) UnpinPath(ctx context.Context, in *api.PinPath, out *api.Pin) error {
```

Comment:
How can unpin utilize pinOptions?

Reply:
it doesn't, but for parallelism (allows using same code to call pin/unpin) and if at some point in history we can unpin with options it's marginally better this way.